decentraland/unity-explorer
# feat: sky controlls panel

## What does this PR change?



Adds a popup to the left sidebar that opens sky controls which allow setting the current time of the skybox (and toggling whether it's dynamic or static).

Note that we currently don't have a hover / click animation for the sidebar button.

## How to test the changes?



1. Launch the explorer
2. Click on the new Sky button (left sidebar, bottom)
3. Make sure the skybox is correctly changed.
4. Make sure that the other buttons in the sidebar behave correctly.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
